docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_table.md

```diff
@@ -126,15 +126,15 @@ In the example above, there are three split points and so four tablets will be c
 - tablet 3: `a=200, b=<lowest>` to `a=200, b=5`
 - tablet 4: `a=200, b=5` to `a=<highest>, b=<highest>`
 
-### COLOCATED
+### COLOCATION
 
 Colocated table support is currently in [Beta](/preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag).
 
 For colocated databases, specify `false` to opt this table out of colocation. This means that the table won't be stored on the same tablet as the rest of the tables for this database, but instead, will have its own set of tablets.
 
-Use this option for large tables that need to be scaled out. See [colocated tables architecture](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/ysql-colocated-tables.md) for more details on when colocation is useful.
+Use this option for large tables that need to be scaled out. See [colocated tables architecture](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/ysql-colocated-tables.md) for more details on when colocation is beneficial.
```

Comment:
Instead of linking to the architecture, maybe we should link to the colocation documentation? @NValsan